.github/workflows/cd.yml

```
@@ -0,0 +1,69 @@
+    steps:
+      - name: Deploy to Azure VM
+        # SSH를 통해 원격으로 명령어를 실행하는 액션
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.VM_HOST }}
+          username: ${{ secrets.VM_USERNAME }}
+          key: ${{ secrets.VM_SSH_PRIVATE_KEY }}
+          script: |
+            # VM에서 ACR에 로그인
+            docker login ${{ secrets.ACR_LOGIN_SERVER }} -u ${{ secrets.ACR_USERNAME }} -p ${{ secrets.ACR_PASSWORD }}
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)